src/vast_tracker.js

```diff
@@ -156,6 +160,24 @@ export class VASTTracker extends EventEmitter {
     this.progress = progress;
   }
 
+  /**
+   * Check if a quertile is reached, and not already triggered
+   *
+   * @param {String} quartile - Quartile name
+   * @param {Number} time - Time offset, when this quartile is reached
+   * @param {Number} progress - Current progress of the ads
```

Comment:
For `time` and `progress` parameters, let's specify if the given time is in seconds or milliseconds